data/changelog.txt

```diff
@@ -479,6 +497,16 @@ Being mute prevents casting VERBAL spells
 [Xedra Evolved] Make Paraclesian fae's magic easier in appropriate terrain
 Aftershock: Basic Hacking EOCs
 [Xedra Evolved] Make Paraclesian fae's magic harder in some circumstances
+Aftershock: Fix several low priority issues.
+Psionic powers say "Power level" instead of "Spell level" in the UI
+[MoM] Update chance to bypass prerequisites when learning psionic powers to use `test_eoc`
+[Xedra Evolved] Update Homullus civilization checker to run once per OMT
+[Xedra Evolved] Update Undine water-checking EoCs to use `test_eoc`
+[Xedra Evolved] Update Arvore forest-checking EoCs to use `test_eoc`
+[MoM] Torrential channeling research no longer has required proficiencies
+Aftershock: Add urban farms as town buildings
+Summon Mojocycle gets better at lvl 10
+[Magiclysm] Add more enchants to effect scaling
```

Comment:
```suggestion
Aftershock: Adds one variant of Habitat Block to the Aftershock Exoplanet. Also adds hackable display cases and doors.
Psionic powers say "Power level" instead of "Spell level" in the UI
[MoM] Torrential channeling research no longer has required proficiencies
Aftershock: Add urban farms as town buildings
Summon Mojocycle gets better at lvl 10
```